scenarioo-client/test/e2e/specs/configure_label_colors.ts

```diff
@@ -46,4 +45,25 @@ useCase('Configure label colors')
 
             });
 
+        scenario('Create a label configuration with a custom color')
+            .it(async () => {
+                const customLabelColor: string = '#00FF00';
+                const numberOfPreconfiguredColors: number = 3;
```

Comment:
auch hier sind die Types unnötig - Type inference

Reply:
Thanks for the info.